docs/snippets/common/component-story-disable-controls.js.mdx

```diff
@@ -5,18 +5,16 @@ import { YourComponent } from './YourComponent';
 
 export default {
   /* 👇 The title prop is optional.
-  * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
-  * to learn how to generate automatic titles
-  */
+   * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
   title: 'YourComponent',
   component: YourComponent,
   argTypes: {
     // foo is the property we want to remove from the UI
     foo: {
-      table: {
-        disable: true,
-      },
+      removeIf: true,
```

Comment:
Does this mean we no longer have to pass diable to the table object?
We will use `removeIf` and `addIf` everywhere?